packages/xo-web/src/xo-app/logs/backup-ng/log-alert-body.js

```diff
@@ -76,6 +105,234 @@ const UNHEALTHY_VDI_CHAIN_ERROR = 'unhealthy VDI chain'
+          {_.keyValue(
+            _('taskMergedDataSpeed'),
+            formatSpeed(task.merge.size, task.merge.duration)
+          )}
+        </div>
+      )}
+      {task.isFull !== undefined &&
+        _.keyValue(_('exportType'), task.isFull ? 'full' : 'delta')}
+    </div>
+  )
+}
+
+const PoolTask = ({ task }) => {
+  const id = task.data.id
+  return (
+    <div>
+      <Pool id={id} link newTab /> ({id.slice(4, 8)}){' '}
+      <TaskStateInfos status={task.status} />
+      <Warnings warnings={task.warnings} />
+      <TaskDate label='taskStart' value={task.start} />
+      <TaskEndAndDuration task={task} />
+      <TaskError task={task} />
+    </div>
+  )
+}
+
+const XoTask = ({ task }) => (
+  <div>
+    <Icon icon='menu-xoa' /> XO <TaskStateInfos status={task.status} />
+    <Warnings warnings={task.warnings} />
+    <TaskDate label='taskStart' value={task.start} />
+    <TaskEndAndDuration task={task} />
+    <TaskError task={task} />
+  </div>
+)
+
+const SnapshotTask = ({ task }) => (
+  <div>
+    <Icon icon='task' /> {_('snapshotVmLabel')}{' '}
+    <TaskStateInfos status={task.status} />
+    <Warnings warnings={task.warnings} />
+    <TaskDate label='taskStart' value={task.start} />
+    {task.end !== undefined && <TaskDate label='taskEnd' value={task.end} />}
+    <TaskError task={task} />
+  </div>
+)
+
+const RemoteTask = ({ children, task }) => {
+  const id = task.data.id
+  return (
+    <div>
+      <Remote id={id} link newTab /> ({id.slice(4, 8)}){' '}
+      <TaskStateInfos status={task.status} />
+      <Warnings warnings={task.warnings} />
+      {children}
+      <TaskDate label='taskStart' value={task.start} />
+      <TaskEndAndDuration task={task} />
+      <TaskError task={task} />
+    </div>
+  )
+}
+
+const SrTask = ({ children, task }) => {
+  const id = task.data.id
+  return (
+    <div>
+      <Sr id={id} link newTab /> ({id.slice(4, 8)}){' '}
+      <TaskStateInfos status={task.status} />
+      <Warnings warnings={task.warnings} />
+      {children}
+      <TaskDate label='taskStart' value={task.start} />
+      <TaskEndAndDuration task={task} />
+      <TaskError task={task} />
+    </div>
+  )
+}
+
+const TransferOrMergeTask = ({ task }) => {
+  const size = get(() => task.result.size)
+  return (
+    <div>
+      <Icon icon='task' /> {task.message}{' '}
+      <TaskStateInfos status={task.status} />
+      <Warnings warnings={task.warnings} />
+      <br />
+      <TaskDate label='taskStart' value={task.start} />
+      <TaskEndAndDuration task={task} />
+      <TaskError task={task} />
+      {size > 0 && (
+        <div>
+          {_.keyValue(_('operationSize'), formatSize(size))}
+          <br />
+          {_.keyValue(
+            _('operationSpeed'),
+            formatSpeed(size, task.end - task.start)
+          )}
+        </div>
+      )}
+    </div>
+  )
+}
+
+const TaskLi = ({ className, task, ...props }) => {
+  const type = get(() => task.data.type)
+  if (type !== undefined) {
```

Comment:
```suggestion
    if (type !== undefined) {
    return (
      <li className={className}>
        {type === 'VM' && <VmTask task={task} {...props} />}
        {type === 'remote' && <RemoteTask task={task} {...props} />}
        {type === 'SR' && <SrTask task={task} {...props} />}
        {type === 'pool' && <PoolTask task={task} {...props} />}
        {type === 'xo' && <XoTask task={task} {...props} />}
        {type === 'VM' && <VmTask task={task} {...props} />}
      </li>
    )
  }
```

Better ?

Reply:
It's better except if a task not match with our cases, an empty item will be displayed.